newsfragments/3779.bugfix

```diff
@@ -0,0 +1 @@
+Fixed bug where share corruption events were not recorded on Windows.
```

Comment:
Can you make it clear here that this is about behavior for a storage server running on Windows (ie, it doesn't matter where the client is running)?